packages/react-server-cli/src/index.js

```diff
@@ -1,3 +1,7 @@
-import startServer from "./startServer"
+require.extensions['.css'] = function(module, filename) {
+  return module._compile("", filename);
+};
```

Comment:
This is still a bit of a hack, but probably the best thing if we don't want to do server-side webpack.

Note that we only include CSS here, which means that this PR will not support JS files that directly `require` a font or image file. This PR only supports fonts and images when they are included in CSS using the `url(foo.png)` syntax.